tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/YugipediaCardTextMapper.java

```diff
@@ -0,0 +1,364 @@
+	protected void updateCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::lore,
+			Card::getCardText,
+			card::setCardText
+		);
+	}
+
+	@AfterMapping
+	protected void updateEsCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::es_lore,
+			c -> c.getLocalizedData().getEs(),
+			cardText -> card.getLocalizedData().setEs(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateDeCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::de_lore,
+			c -> c.getLocalizedData().getDe(),
+			cardText -> card.getLocalizedData().setDe(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateFrCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::fr_lore,
+			c -> c.getLocalizedData().getFr(),
+			cardText -> card.getLocalizedData().setFr(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateItCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::it_lore,
+			c -> c.getLocalizedData().getIt(),
+			cardText -> card.getLocalizedData().setIt(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateJaCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::ja_lore,
+			c -> c.getLocalizedData().getJa(),
+			cardText -> card.getLocalizedData().setJa(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateKoCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::ko_lore,
+			c -> c.getLocalizedData().getKo(),
+			cardText -> card.getLocalizedData().setKo(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updatePtCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			cardTable2::pt_lore,
+			c -> c.getLocalizedData().getPt(),
+			cardText -> card.getLocalizedData().setPt(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateZhHansCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			() -> getZhLore(cardTable2),
+			c -> c.getLocalizedData().getZhHans(),
+			cardText -> card.getLocalizedData().setZhHans(cardText)
+		);
+	}
+
+	@AfterMapping
+	protected void updateZhHantCardText(
+		final CardTable2 cardTable2,
+		@MappingTarget final Card card
+	) {
+		updateCardTextWithFullLore(
+			card,
+			() -> getZhLore(cardTable2),
+			c -> c.getLocalizedData().getZhHant(),
+			cardText -> card.getLocalizedData().setZhHant(cardText)
+		);
+	}
+
+	private MarkupString getZhLore(final CardTable2 cardTable2) {
```

Comment:
This should be removed and chinese lore being handled with the separation between traditional and simplified.